Assets/Scripts/UIShopElement.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class UIShopElement : MonoBehaviour
+{
+   public int id;
+   public int price;
+   public Text priceTxt; 
+   public Text idTxt;
+   public Button purchaseBtn;
+   
+   public Button sellBtn;
+   public void Awake(){
+    
+    purchaseBtn.onClick.AddListener(OnPurchase);
+    sellBtn.onClick.AddListener(OnSale);
+   UpdateView();
+   }
+    
+    public void SetData(int id){
+        this.id = id;
+        price = id*10;
```

Comment:
This script contains values obtained randomly. We should not push until we include a proper game logic. 
Please remind us to talk about the star's price on next meeting. 
We should maybe link the price to the size of the star?